src/ontology/edits/oeo-physical.omn

```diff
@@ -6709,6 +6716,18 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/932",
         OEO_00000533 some OEO_00000139
     
     
+Class: OEO_00240015
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Liquid air production is a production that produces liquid air by cooling and compressing (gaseous) air."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/937
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/945",
+        rdfs:label "liquid air production"
+    
+    SubClassOf: 
+        OEO_00240003
```

Comment:
I think, we should add here the axiom `'has physical output' some 'liquid air'`.

Reply:
Should we also add: `'has physical input' some air`?

Reply:
Good idea.